src/ObjCRuntime/RuntimeOptions.cs

```diff
@@ -67,6 +67,9 @@ static string ParseHttpMessageHandler (string value)
 			case null:
 				return HttpClientHandlerValue;
 			case HttpClientHandlerValue:
+				if (Driver.App.Platform == Utils.ApplePlatform.WatchOS)
+					throw ErrorHelper.CreateError (2015, "Invalid HttpMessageHandler `{0}` for watchOS. Valid values are NSUrlSessionHandler (default) or CFNetworkHandler", value);
```

Comment:
`or CFNetworkHandler` ? I don't think CFNetwork is (public) available on watchOS 2.x